### ISSUE TITLE
[BITV] Add title to Avatar component

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#35857]
## Problem

To show a tooltip on a avatar we currently use v-tooltip. An example can be found in https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/NcAvatar/NcAvatar.vue#L112. Another example is the `files_sharing` app: https://github.com/nextcloud/server/issues/35654. Example here: https://github.com/nextcloud/server/blob/master/apps/files_sharing/src/components/SharingEntry.vue#L29

v-tooltip is not accessible and we want to migrate to native tooltips (title attribute) instead. An overview can be found in https://github.com/nextcloud/nextcloud-vue/issues/2503.

## Solution

Add a `title` prop to the Avatar component that will render as `title` attribute on the HTML element.
